<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgWaveletDecompose</class>
 <widget class="QWidget" name="WdgWaveletDecompose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of wavelet detail scales:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="waveletScaleSpinbox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="setLayerCheckbox">
     <property name="text">
      <string>Set layer mode for recomposition</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="newImageCheckbox">
     <property name="text">
      <string>Create new image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
